spec/features/programs_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'spec_helper'
 
-describe '番組情報ページ' do
+describe '放送予定ページ' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.